ryu/services/protocols/ovsdb/event.py

```diff
@@ -13,16 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ryu.controller import event
+from ryu.controller import event as ryu_event
 from ryu.controller import handler
 
 
-class EventRowBase(event.EventBase):
-    def __init__(self, system_id, table, row):
-        super(event.EventBase, self).__init__()
+class EventRowBase(ryu_event.EventBase):
+    def __init__(self, system_id, table, row, event_type):
+        super(EventRowBase, self).__init__()
         self.system_id = system_id
         self.table = table
         self.row = row
+        self.event_type = event_type
```

Comment:
The strings are so we can use them for concatenation to form the name of the row level events from the generic crud events.